integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/hive/CatalogHiveIT.java

```diff
@@ -1126,6 +1126,31 @@ void testDropAndRename() {
         NoSuchMetalakeException.class, () -> client.loadMetalake(NameIdentifier.of(metalakeName1)));
   }
 
+  @Test
+  public void testDropHiveManagedTable() throws TException, InterruptedException, IOException {
```

Comment:
Can you please also add the `testDropHiveExternalTable` method?

Reply:
added, I also add a check when drop external tables.